### PR TITLE
Single Wasm bundle

### PR DESCRIPTION
**Pull request by jprochazk**
### What

- Remove all traces of `re_viewer_debug`

This gives us more time to deal with the `crates.io` upload size limit issue:
image

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/3580) (if applicable)

- [PR Build Summary](https://build.rerun.io/pr/3580)
- [Docs preview](https://rerun.io/preview/dbd0591769c813229523a9628e2770fd59cb8634/docs) 
- [Examples preview](https://rerun.io/preview/dbd0591769c813229523a9628e2770fd59cb8634/examples) 
- [Recent benchmark results](https://ref.rerun.io/dev/bench/)
- [Wasm size tracking](https://ref.rerun.io/dev/sizes/)

### CODE REVIEW

File: scripts/ci/sizes.py [hunk 1]
```diff
@@ -9,12 +9,10 @@
...
 
     python3 scripts/ci/sizes.py measure \
-        "Wasm (release)":web_viewer/re_viewer_bg.wasm \
-        "Wasm (debug)":web_viewer/re_viewer_debug_bg.wasm
+        "Wasm (release)":web_viewer/re_viewer_bg.wasm
 
     python3 scripts/ci/sizes.py measure --format=github \
-        "Wasm (release)":web_viewer/re_viewer_bg.wasm \
-        "Wasm (debug)":web_viewer/re_viewer_debug_bg.wasm
+        "Wasm (release)":web_viewer/re_viewer_bg.wasm
```
Comment (emilk) [inline scripts/ci/sizes.py:15]: This is a bit misleading, since we can't know if `re_viewer_bg.wasm` is the release or debug build here

Comment (emilk) [reply scripts/ci/sizes.py:15]: Make sure the CI sizes.py is measuring the right wasm

Comment (jprochazk) [reply scripts/ci/sizes.py]: The CI up until now was measuring the release size, so it should be fine

Comment (jprochazk) [reply scripts/ci/sizes.py]: image

Comment (jprochazk) [reply scripts/ci/sizes.py]: the `web_viewer` artifact it downloads is this one from `reusable_build_web`
image

**Review by emilk — APPROVED** [review]

**Pull request merged**